### PR TITLE
Defaults Play button to first unwatched Episode

### PR DESCRIPTION
**Pull request by KingLucius**

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt [hunk 1]
```diff
@@ -782,7 +783,7 @@ class ResultFragmentTv : Fragment() {
                 //    resultEpisodeLoading.isVisible = episodes is Resource.Loading
                 if (episodes is Resource.Success) {
 
-                    val lastWatchedIndex = episodes.value.indexOfLast { ep -> ep.videoWatchState == VideoWatchState.Watched }
+                    val lastWatchedIndex = episodes.value.indexOfLast { ep -> ep.getWatchProgress() >= (NEXT_WATCH_EPISODE_PERCENTAGE / 100).toFloat() }
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt:786]: (90 / 100).toFloat() ? Wont that just be zero due to int rounding?

Comment (fire-light42) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt:786]: I think you mean to do NEXT_WATCH_EPISODE_PERCENTAGE.toFloat() / 100.0f

Comment (KingLucius) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt:786]: > (90 / 100).toFloat() ? Wont that just be zero due to int rounding?

That would be 0.90f

Comment (KingLucius) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt:786]: > I think you mean to do NEXT_WATCH_EPISODE_PERCENTAGE.toFloat() / 100.0f

Same i think , i will double check again

**Pull request merged**